jenkins/bake_hostclass.sh

```diff
@@ -10,7 +10,11 @@ echo "hostclass: $HOSTCLASS"
 
 source "${SELF_DIR}/boto_init.sh"
 
-disco_bake.py --debug bake --hostclass $HOSTCLASS --use-local-ip
+if [ "${BAKE_TO_STAGE}" != "" ]; then
+    stage_arg="--stage=$BAKE_TO_STAGE"
+fi
+
+disco_bake.py --debug bake --hostclass $HOSTCLASS --use-local-ip $stage_arg
```

Comment:
The command is:

`disco_bake.py bake --hostclass HOSTCLASS --stage STAGE --use-local-ip`

Why do we need this `if` statement? Can't we pass $STAGE dhe same way we pass $HOSTCLASS?